lib/rules/at-rule-property-required-list/index.js

```diff
@@ -30,39 +30,62 @@ const rule = (primary) => {
 			return;
 		}
 
+		/** @type {Map<string, Set<string>|undefined>} */
+		const propLists = new Map();
+
+		for (const key in primary) {
+			if (!Object.hasOwnProperty.call(primary, key)) continue;
+
+			const propList = flattenArray(primary[key]);
```

Comment:
[suggest] How about reducing the loop count by using `for...of` and `Object.entries()`?

```suggestion
		for (const [key, value] of Object.entries(primary)) {
			const propList = flattenArray(value);
```

Reply:
Good call